low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -47,7 +47,8 @@ def __init__(self):
         self.server_frus = {
             'cpu': self.get_cpu_info,
             'platform_sensors': self.get_platform_sensors_info,
-            'memory': self.get_mem_info
+            'memory': self.get_mem_info,
+            'fans': self.get_fanmodules_info
         }
         self._ipmi = IpmiFactory().get_implementor("ipmitool")
```

Comment:
Before this line, we must have validate_server_type_support() like we added in storage.py
Please refer generate resource health view file as well for this.

Reply:
Added server_type validation.
Please review now